src/main/webapp/js/userMap.js

```diff
@@ -31,8 +31,13 @@ function handleData(err, countryCoordinates, userData) {
         }
     }
 
-    d3.select('#totalCount').text(userData.length + ' institutions from ' + countriesArr.length + ' countries');
-
+    //set the institution count in the page
+    var e = document.getElementById('totalUserCount');
+    e.innerHTML=userData.length;
```

Comment:
I think you can skip the variable `e` (and similarly `f` down there), just lump it together (`document.getElementById.innerHTML = ...`)